oxidation/client/src/locales/en-US.json

```
@@ -165,6 +193,13 @@
             "lastLoginMinutes": "one minute ago | {minutes} minutes ago",
             "lastLoginHours": "one hour ago | {hours} hours ago",
             "lastLoginDays": "one day ago | {days} days ago"
+        },
+        "filesize": {
+            "bytes": "B",
+            "kilobytes": "KB",
+            "megabytes": "MB",
+            "gigabytes": "GB",
+            "terabytes": "TB"
```

In `filesize.ts` the System use the value in $1024^n = 2^{10n}$ notation.

I suggest to use `iB` to prevent confusion between `bit` & `bytes` $8_{bit} = 1_{bytes}$ 

```suggestion
            "bytes": "Bit",
            "kilobytes": "KiB",
            "megabytes": "MiB",
            "gigabytes": "GiB",
            "terabytes": "TiB"
```

More information about [Binary prefix](https://en.wikipedia.org/wiki/Binary_prefix)

---

Taken as is from the current GUI (https://github.com/Scille/parsec-cloud/blob/master/parsec/core/gui/file_size.py), I don't remember why we chose this format but we did discuss it at length.